### PR TITLE
Fix/text formatting

### PR DESCRIPTION
**Pull request by pinocchio-life-like**

### CODE REVIEW

File: packages/app/components/card/CustomCardHeader.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const CustomCardHeader = ({ data, title, link, actionsComponent }) => {
       }
       </RStack>
       <View>
-        <Link href={`/profile/${data.owner_id.id || data.owner_id}`}>
+        <Link href={`/profile/${data.owner_id.id || data.owner_id}`} style={{ textDecoration: 'none' }}>
```
Comment (andrew-bierman) [inline packages/app/components/card/CustomCardHeader.tsx:24]: We should probably add this to the Link we export

Comment (pinocchio-life-like) [reply packages/app/components/card/CustomCardHeader.tsx]: > We should probably add this to the Link we export

@andrew-bierman Yes, but the Link is getting imported from crosspath as a function declaration not a component, and is even in .d.ts file.

So Instead I have defined the same Link component inside ui directory and Imported the Link from crosspath file and set the style property on it now, and imported that for usage. I hope that is okay. lmk please!

Comment (andrew-bierman) [reply packages/app/components/card/CustomCardHeader.tsx]: Got it, yes that is a good solution. We should treat link as a UI component I believe, considering it does apply styles

**Comment by jeweltee** [comment]
@pinocchio-life-like the update looks good! Thank you

**Review by andrew-bierman — APPROVED** [review]

**Review by jeweltee — COMMENTED** [review]
Update looks good

**Pull request merged**